### PR TITLE
font-ubuntu-ttf: Inclusion of homepage and metainfo.xml

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Change to new source (old one no longer works)
- Adding `homepage` key to `package.yml`( Part of #411)
- Including `metainfo.xml` (Part of #449)

**Test Plan**

- Build the package
- Run `appstream-builder --packages-dir=. --include-failed -v`
- Check that the package doesn't show up in the `example-failed.xml.gz`
- Verified the added homepage loads information about the package

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/f/font-ubuntu-ttf/files/ubuntu.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Solus Developers <copyright@getsol.us> -->
+<component type="font">
+  <id>Ubuntu</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Ubuntu</name>
+  <homepage>https://design.ubuntu.com/font</homepage>
```
Comment (TraceyC77) [inline packages/f/font-ubuntu-ttf/files/ubuntu.metainfo.xml:7]: In this file the format of the homepage needs to be in this format:

```
<url type="homepage">https://design.ubuntu.com/font</url>
```

File: packages/f/font-ubuntu-ttf/files/ubuntu.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Solus Developers <copyright@getsol.us> -->
+<component type="font">
+  <id>Ubuntu</id>
```
Comment (TraceyC77) [inline packages/f/font-ubuntu-ttf/files/ubuntu.metainfo.xml:4]: This should match the name of the package, in this case "font-ubuntu-ttf"

**Review by TraceyC77 — CHANGES_REQUESTED** [review]
Just a couple of minor things and then this can be merged. Thanks!

File: packages/f/font-ubuntu-ttf/files/ubuntu.metainfo.xml (hunk 1 above)
Comment (malfisya) [reply packages/f/font-ubuntu-ttf/files/ubuntu.metainfo.xml]: done

File: packages/f/font-ubuntu-ttf/files/ubuntu.metainfo.xml (hunk 2 above)
Comment (malfisya) [reply packages/f/font-ubuntu-ttf/files/ubuntu.metainfo.xml]: done

**Review by TraceyC77 — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**